shuffle_sdk/shuffle_sdk.py

```diff
@@ -370,7 +370,7 @@ def __init__(self, redis=None, logger=None, console_logger=None):#, docker_clien
             action_id = self.action.get("id")
             fullKey = str(self.current_execution_id) + "-" + str(action_id)
             if fullKey in pastAppExecutions:
-                self.logger.info(f"[INFO] Duplicate execution detected for execution id - {self.current_execution_id} and action - {self.action["id"]}")
+                self.logger.error(f"[ERROR] Duplicate execution detected for execution id - {self.current_execution_id} and action - {self.action["id"]}")
```

Comment:
One more thing:
- Since you're using `action_id` above, why do you use self.action["id"] below? self.action.get("id") is safe, but self.action["id"] is not

This may lead to it continuing due to action ID failing to load in the 2nd one 😆